test/teos/unit/test_appointments_dbm.py

```diff
@@ -18,7 +18,7 @@
 
 
 @pytest.fixture(scope="module")
-def watcher_appointments():
+def watcher_appointments(run_bitcoind):
```

Comment:
We should replace the `generate_dummy_tracker fixture` to call `generate_dummy_tracker`

Reply:
Done in 3d016331a1a94d4eade971a6f1955c4ff23294dc, and done a heavier refactoring of both the functions (that are now fixtures) in bcea9473c5b7a34d74f1c2631f22d534f2922f8c.